Source/WebKit/UIProcess/API/glib/WebKitError.cpp

```diff
@@ -112,12 +112,30 @@ GQuark webkit_snapshot_error_quark()
     return g_quark_from_static_string("WebKitSnapshotError");
 }
 
+#if ENABLE(2022_GLIB_API)
+/**
+ * webkit_web_extension_error_quark:
+ *
+ * Gets the quark for the domain of Web Extension errors.
+ *
+ * Returns: web extension error domain.
+ * 
+ * Since: 2.52
+ */
+GQuark webkit_web_extension_error_quark()
+{
+    return g_quark_from_static_string("WebKitWebExtensionError");
+}
+#endif
+
 /**
  * webkit_web_extension_match_pattern_error_quark:
  *
  * Gets the quark for the domain of Web Extension Match Pattern errors.
  *
  * Returns: web extension match pattern error domain.
+ * 
+ * Since: 2.48
```

Comment:
`2.52`

Reply:
I'll set this to 2.52, but the actual domain here has existed since 2.48. If anything it's a bug that this wasn't added in originally.

Reply:
No, set it to 2.48. Patrick surely just didn't notice.